Functions/xASL_test_GetLogContent.m

```diff
@@ -0,0 +1,308 @@
+function [logContent] = xASL_test_GetLogContent(rootDir, printContent, storeRelativePath, exportTable)
```

Comment:
For a broader discussion. 
We have _ut_ for unit tests and _test_ for other tests? Should we have these two or unify under a single prefix. Or it doesn't make sense as this is not really part of the unit testing?

Reply:
Good idea to discuss this elsewhere;)